roles/kolibri/tasks/main.yml

```diff
@@ -60,14 +68,16 @@
     state: stopped
 
 
-- name: Run Kolibri migrations to begin populating {{ kolibri_home }}    # i.e. /library/kolibri
-  shell: export KOLIBRI_HOME="{{ kolibri_home }}" && "{{ kolibri_exec_path }}" manage migrate
-  ignore_errors: yes
-  become: yes
-  become_user: "{{ kolibri_user }}"
-  when: kolibri_provision | bool
+# 2019-10-01: Should no longer be nec, thanks to /etc/kolibri/daemon.conf
+#             containing KOLIBRI_HOME="/library/kolibri" (above)
+#- name: Run Kolibri migrations to begin populating {{ kolibri_home }}    # i.e. /library/kolibri
+#  shell: export KOLIBRI_HOME="{{ kolibri_home }}" && "{{ kolibri_exec_path }}" manage migrate
+#  ignore_errors: yes
+#  become: yes
+#  become_user: "{{ kolibri_user }}"
+#  when: kolibri_provision | bool
```

Comment:
It's true that this isn't necessary because kolibri is started after installing the .deb -- this causes the database to be migrated. But I think you should call `systemctl stop kolibri` to ensure it has started and stopped fully before running `provisiondevice`.

Reply:
Great!

It seems you didn't notice that `systemctl stop kolibri` is happening just above in lines:
https://github.com/holta/iiab/blob/kolibri-belt-tightening/roles/kolibri/tasks/main.yml#L63-L68

And then is restarted in lines:
https://github.com/holta/iiab/blob/kolibri-belt-tightening/roles/kolibri/tasks/main.yml#L114-L118

_So I think we're all covered!_